docparse/aryn_sdk.mdx

```diff
@@ -69,4 +69,34 @@ with open("mypresentation.ppt", "rb") as f:
    data = partition_file(f)
 ```
 
+## Chunking a document
+
+Chunking support has been added in v0.1.9. You can enable the default chunking options by specifying an empty dict:
+
+```python
+from aryn_sdk.partition import partition_file
+with open("mydocument.pdf", "rb") as f:
+   data = partition_file(f, chunking_options={})
+```
+
+Here is an example specifying certain chunking options:
+
+```python
+from aryn_sdk.partition import partition_file
+with open("mydocument.pdf", "rb") as f:
+   data = partition_file(f, 
+      chunking_options={
+         "merging_strategy": "header_augmenter",
```

Comment:
are we using the "header_augmenter" still? I thought we got rid of that name

Reply:
Good catch! This should be `"strategy": "context_rich"`. I've now written a fix in #22.